### ISSUE TITLE
python3 RuntimeWarning

### ISSUE THREAD

**Issue by GitMensch** [zaufi/pluggable-output-processor#15]
```
/usr/lib/python3.9/subprocess.py:941: RuntimeWarning: line buffering (buffering=1) isn't supported in binary mode, the default buffer size will be used
  self.stdout = io.open(c2pread, 'rb', bufsize)
```

and because that's a warning the actual formatting still works

**Comment by zaufi** [comment]
@GitMensch ,

Thank you for your interest and all issues opened! However, nowadays, as you may see ;) this project is a bit abandoned^W out of date :) Meanwhile, I still like it, and some day definitely will modernize it and update it for modern compilers (there are plenty of ideas about it in my mind %)

**Comment by GitMensch** [comment]
Maybe fixing the "easy" ones like this, #17, #19  and including the additional line proposed in #18 plus an "release" to pip would be a very good intermediate step :-)
I found it to be "nearly working well enough", those above would solve the missing gap to suggest our user-base to use it, which is likely much faster than handling it in the application (and would also work for old versions).

Of course other possible FRs come to mind (for [possibly much] later) like "convert to JSON/XML format to let it be easily used by CI/CD or IDEs" and the option to "stack plugins".

**Comment by zaufi** [comment]
> Of course other possible FRs come to mind (for [possibly much] later) like "convert to JSON/XML format to let it be easily used by CI/CD or IDEs" and the option to "stack plugins".

Yeah, I also have ideas about producing JSON and other CI-friendly (e.g., GHA-friendly) service messages to beautify output for CI (grouping messages for example) so it'll be not about only colorizing output :)
